backend/internal/handlers/venue_ratings/venue_ratings.go

```diff
@@ -0,0 +1,34 @@
+package venueratings
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/google/uuid"
+)
+
+func (s *Service) GetAllVenueRatings(c *fiber.Ctx) error {
+	fmt.Println("Pinging correct service")
+	venueID := c.Params("venueid")
+	if venueID == "" {
+		c.Status(http.StatusBadRequest)
+		return fiber.NewError(400)
+	}
+	VenueIDFormatted, err := uuid.Parse(venueID)
+
+	if err != nil {
```

Comment:
Codifying earlier conversation here: Please update to parse request body instead of relying on URL encoding